push-messaging-and-notifications/main.js

```diff
@@ -1,23 +1,42 @@
 'use strict';
 
-var API_KEY = '<YOUR API KEY>';
+var API_KEY = window.GoogleSamples.Config.gcmAPIKey;
+var GCM_ENDPOINT = 'https://android.googleapis.com/gcm/send';
 
 var curlCommandDiv = document.querySelector('.js-curl-command');
 var isPushEnabled = false;
 
 function sendSubscriptionToServer(subscription) {
-  // TODO: Send the subscription.subscriptionId and 
-  // subscription.endpoint to your server and save 
-  // it to send a push message at a later date
+  // TODO: Send the subscription.endpoint
```

Comment:
Except in M43 you'll need to concatenate `subscription.endpoint` and `subscription.subscriptionId` into a single string, and then send that along. Could you create a helper function that handles the concatenation if it's needed and call it here, just to show developers what they'd need to do if they were to implement this function?